text/0112-compress-state-response-message-in-state-sync.md

```diff
@@ -0,0 +1,70 @@
+# RFC-0112: Compress the State Response Message in State Sync
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 14 August 2024                                                                              |
+| **Description** | Compress the state response message to reduce the data transfer during the state syncing    |
+| **Authors**     | Liu-Cheng Xu                                                                                |
+
+## Summary
+
+This RFC proposes compressing the state response message during the state syncing process to reduce the amount of data transferred.
+
+## Motivation
+
+State syncing can require downloading several gigabytes of data, particularly for blockchains with large state sizes, such as Astar, which
+has a state size exceeding 5 GiB (https://github.com/AstarNetwork/Astar/issues/1110). This presents a significant
+challenge for nodes with slower network connections. Additionally, the current state sync implementation lacks a persistence feature (https://github.com/paritytech/polkadot-sdk/issues/4),
+meaning any network disruption forces the node to re-download the entire state, making the process even more difficult.
+
+## Stakeholders
+
+This RFC benefits all projects utilizing the Substrate framework, specifically in improving the efficiency of state syncing.
+
+- Node Operators.
+- Substrate Users.
+
+## Explanation
```

Comment:
I think this requires some implementation details, like what compression algorithm is going to be used